dataactcore/scripts/loadDUNS.py

```diff
@@ -0,0 +1,322 @@
+                                load_duns_by_row(add_data, sess, models, activated_models, benchmarks=benchmarks)
+                        if not update_delete_data.empty:
+                            models, activated_models = get_relevant_models(update_delete_data, benchmarks=benchmarks)
+                            logger.info("loading update_delete data ({} rows)".format(len(update_delete_data.index)))
+                            load_duns_by_row(update_delete_data, sess, models, activated_models, benchmarks=benchmarks)
+                    sess.commit()
+
+            added_rows += nrows
+            batch += 1
+            logger.info('%s DUNS records inserted', added_rows)
+        if benchmarks:
+            logger.info("Parsing {} took {} seconds with {} rows".format(dat_file_name, time.time()-parse_start_time,
+                                                                         added_rows))
+
+
+def process_from_dir(root_dir, file_name, sess, local, monthly=False, benchmarks=False):
+    file_path = os.path.join(root_dir, file_name)
+    if not local:
+        logger.info("Pulling {}".format(file_name))
+        with open(file_path, "wb") as zip_file:
+            sftp.getfo(''.join([REMOTE_SAM_DIR, '/', file_name]), zip_file)
+    parse_sam_file(file_path, sess, monthly=monthly, benchmarks=benchmarks)
+    if not local:
+        os.remove(file_path)
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(description="Get the latest data from SAM and update duns table. By default, it "
+                                                 "loads the latest daily file.")
+    parser.add_argument("--historic", "-i", action="store_true", help='load the oldest monthly zip and all the daily'
+                                                                      'files afterwards from the directory.')
+    parser.add_argument("--local", "-l", type=str, default=None, help='work from a local directory')
+    parser.add_argument("--monthly", "-m", type=str, default=None, help='load a local monthly file')
+    parser.add_argument("--daily", "-d", type=str, default=None, help='load a local daily file')
+    parser.add_argument("--benchmarks", "-b", action="store_true",
+                        help='log times of operations for testing')
+    parser.add_argument("--update", "-u", action="store_true",
+                        help='Run all daily files since latest last_sam_mod_date in table')
+    return parser
+
+
+if __name__ == '__main__':
+    parser = get_parser()
+    args = parser.parse_args()
+
+    historic = args.historic
+    local = args.local
+    monthly = args.monthly
+    daily = args.daily
+    benchmarks = args.benchmarks
+    update = args.update
+
+    with create_app().app_context():
+        configure_logging()
+        sess = GlobalDB.db().session
+
+        if monthly and daily:
+            print("For loading a single local file, you must provide either monthly or daily.")
```

Comment:
We don't want print statements. The logger should be sufficient. (apply to all below print statements)